### PR TITLE
use nodeReuse:false in build.sh and build-test.sh.

### LINKED ISSUE #25030: build-test.sh failing (MSB4062: The "Microsoft.CodeAnalysis.BuildTasks.Csc" task could not be loaded... Assembly with same name is already loaded)
I updated to tip last night.

Ubuntu 16.04 (previously working)

```
./build.sh
./build-test.sh
```
failed.
```
git clean -xdf
./build.sh
./build-test.sh
```
failed.
435 errors building managed assemblies.
```
./build-test.sh generatelayoutonly
```
succeeded (I didn't need the managed test assemblies....

### PR DESCRIPTION
**Pull request by jashook**
This brings `build.sh` and `build-test.sh` further in line with `build.cmd` and `build-test.cmd`. Fixes #25030. Thanks @elinor-fung for the suggested fix!

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -970,7 +970,7 @@ while :; do
     shift
 done
 
-__CommonMSBuildArgs="/p:__BuildArch=$__BuildArch /p:__BuildType=$__BuildType /p:__BuildOS=$__BuildOS $__OfficialBuildIdArg $__SignTypeArg $__SkipRestoreArg"
+__CommonMSBuildArgs="/p:__BuildArch=$__BuildArch /p:__BuildType=$__BuildType /p:__BuildOS=$__BuildOS "/nodeReuse:false" $__OfficialBuildIdArg $__SignTypeArg $__SkipRestoreArg"
```
Comment (elinor-fung) [inline build.sh]: Does this one actually need the quotes?

Comment (jashook) [reply build.sh]: It is not necessary will change.

**Comment by mjsabby** [comment]
@jashook Could you please squash and merge your commit instead of creating a merge commit. It keeps the git history clean and friendly for binary searching the commits. Thank you.

**Comment by jashook** [comment]
@mjsabby thanks for the suggestion. I normally am set up for that, but switching machines has left breadcrumbs of things to setup. Because I  assume I squash and merge I probably would not have caught that for some time. So I appreciate the heads up and have made the change :).

**Pull request merged**